packages/custom-webpack/examples/full-cycle-app/angular.json

```diff
@@ -50,7 +53,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
Any reason why this was removed?

Reply:
(re: https://github.com/just-jeb/angular-builders/issues/944#issuecomment-863207093)